core/src/main/scala/org/http4s/QValue.scala

```diff
@@ -98,10 +101,29 @@ object QValue {
   def unsafeFromString(s: String): QValue =
     fromString(s).fold(throw _, identity)
 
+  private[http4s] val parser: Parser[QValue] = {
+    import cats.parse.Parser.{char => ch, _}
+    import cats.parse.Rfc5234._
+
+    val optWS = (crlf.rep.with1 ~ wsp.rep1).rep
+
+    val qValue = string1(ch('0') ~ (ch('.') ~ digit.rep1).rep)
```

Comment:
`*>` or `<*` instead of `~` if you're not using the tuple you build.

Reply:
Done